tests/integration/suite/actors/reminders/scheduler/remove.go

```diff
@@ -114,8 +114,8 @@ func (r *remove) Run(t *testing.T, ctx context.Context) {
 		DialTimeout: 5 * time.Second,
 	})
 
-	// Use "path/filepath" import, it is using OS specific path separator unlike "path"
-	etcdKeysPrefix := filepath.Join("dapr", "jobs")
+	// Use "path" import, to have the same path separator across OS
```

Comment:
Comment no longer relevant

Reply:
ugh its because I accepted the UI suggestion and batched it. let me fix that bit